src/AutoComplete/styledAutoCompleteComponents.js

```diff
@@ -0,0 +1,35 @@
+import StyledAutoComplete from './StyledAutoComplete';
+import StyledAutoCompleteClearIndicator from './StyledAutoCompleteClearIndicator';
+import StyledAutoCompleteContainer from './StyledAutoCompleteContainer';
+import StyledAutoCompleteControl from './StyledAutoCompleteControl';
+import StyledAutoCompleteDropdownIndicator from './StyledAutoCompleteDropdownIndicator';
+import StyledAutoCompleteIndicatorsContainer from './StyledAutoCompleteIndicatorsContainer';
+import StyledAutoCompleteInput from './StyledAutoCompleteInput';
+import StyledAutoCompleteMenu from './StyledAutoCompleteMenu';
+import StyledAutoCompleteMenuList from './StyledAutoCompleteMenuList';
+import StyledAutoCompleteMultiValue from './StyledAutoCompleteMultiValue';
+import StyledAutoCompleteNoOptionsMessage from './StyledAutoCompleteNoOptionsMessage';
+import StyledAutoCompletePlaceholder from './StyledAutoCompletePlaceholder';
+import StyledAutoCompleteOption from './StyledAutoCompleteOption';
+import StyledAutoCompleteValueContainer from './StyledAutoCompleteValueContainer';
+
+export const subComponents = {
```

Comment:
this is a separate export because StyledAutoComplete needs the rest of these, but we need these plus StyledAutoComplete itself as far as exporting from the module. Doing this prevents StyledAutoComplete from trying to import itself.